meetings/tool/src/main/resources/card _eu.properties

```diff
@@ -1,18 +1,19 @@
-decription_link_text=View Description
```

Comment:
Name file should be card_eu.properties. There is a blank space before _

Reply:
I think he fixed it!